contents/docs/feature-flags/common-questions.mdx

```diff
@@ -80,10 +82,13 @@ Behavioral cohorts are [dynamic cohorts](/docs/data/cohorts#dynamic-cohorts) cre
 Behavioral cohorts are dynamic cohorts created using the options under the "behavioral" subheading.
 </Caption>
 
-Feature flags require fast evaluation and behavioral queries are relatively slow. Thus enabling these queries would significantly impact the performance of feature flags. 
+We don't support using them with feature flags because querying them is relatively slow. Feature flags require fast evaluation and these queries would significantly impact the evaluation speed of the entire service. 
```

Comment:
This is something the flags team is going to build this quarter, so I can't wait to update these docs :)